app/assets/locale/es.json

```diff
@@ -143,67 +143,67 @@
-		"large-text": "¡Ella no es autista, deja de&nbsp;decir&nbsp;eso!",
-		"sub-text": "Las personas autistas a menudo tienen problemas para comunicarse, mantenerse en silencio en su propio rincón todo el tiempo, Satania&nbsp;no es así en absoluto.<br><b>¡Probablemente ella es menos autista que tu!</b>",
-		"small-text": "Y aunque ella actuó como un niño pequeño en ese episodio de navidad, <br>¡fue por que raphi le hizo hacerlo!"
+		"large-text": "Por cierto, ella no es autista, deja de&nbsp;decir&nbsp;eso",
+		"sub-text": "Las personas autistas a menudo tienen problemas para comunicarse, manteniendose callado en su propio rincón todo el tiempo, Satania&nbsp;no es así del todo.<br><b>¡Probablemente ella es menos autista que tu!</b>",
+		"small-text": "Y aunque ella actuó como un niño pequeño en ese episodio de navidad, <br>¡Fue por que Raphiel le hizo hacerlo!"
 	},
 	"pope-francis": {
 		"large-text": "Incluso el Papa Francisco ama a Satania",
-		"sub-text": "\"Satania es mi waifu numero dos, el numero uno es Jesus por&nbsp;supuesto!<br>Yo la bendigo y a todos sus seguidores!\""
+		"sub-text": "\"Satania es mi waifu numero dos, el numero uno es ¡Jesus por&nbsp;supuesto!<br>¡Yo la bendigo y a todos sus seguidores!\""
 	},
 	"flipping-cards": {
 		"large-text": "Ella reemplaza a todas las otras waifus del show",
 		"sub-text": "Cuando lo piensas, Satania es una alternativa perfecta a cada personaje de Gabriel Dropout... <span class=\"small\">Haga click en cualquiera de las tarjetas para ver las respuestas</span>",
 		"cards": {
 			"gabriel": {
-				"question": "Necesitas una chica <b>poco amable, que no le importa nadie más</b> y piensa que sus pasatiempos son más importantes que todo?",
-				"answer": "Entonces tu necesitas al demonio que <b>no le importan los humanos ni los angeles y siempre esta tratando de ser malvada</b>, en vez de estudiar o otra cosa... <br> Ademas, <b>ella no hace su tarea a PROPOSITO!!!</b>"
+				"question": "¿Necesitas una chica <b>poco amable, que no le importa nadie más</b> y piensa que sus pasatiempos son más importantes que todo?",
+				"answer": "Entonces tu necesitas al demonio que <b>no le importan los humanos ni los angeles y siempre esta tratando de ser malvada</b>, en vez de estudiar o otra cosa... <br> Ademas, <b>ella no hace su tarea ¡¡¡A PROPOSITO!!!</b>"
 			},
 			"vigne": {
-				"question": "Te gustaria una <b>linda demonio moe</b> que <b>quiere ser malvada, pero es mala en ello</b>, terminando haciendo haciendo cosas buenas como enseñarle a Gabriel? ",
-				"answer": "Entonces probablemente te gustara <b>esta demonio que hace su mayor esfuerzo para ser la mejor demonio </b>, pero es frecuentemente manipulada a <b>hacer cosas buenas</b>, incluso aunque ella hace lo mejor para enseñar a Gabriel buenas lecciones..."
+				"question": "¿Te gustaria una <b>linda demonio moe</b> que <b>quiere ser malvada, pero es mala en ello</b>, terminando haciendo haciendo cosas buenas como enseñarle a Gabriel? ",
```

Comment:
SP: Eso no lo he visto, eso es un error del archivo sin editar si miras bien, yo lo único que hice fue poner el "¿" al principio. Pero si, es mejor borrar el "haciendo" repetido.
EN: I didn't saw it, thats an error of the original file if you look closely, the only thing i did is putting the "¿" at the beggining. But yeah, it will be better to deleted the repeated "haciendo".